### DISCUSSION TITLE
cmp+documentation size problem in macos

### DISCUSSION THREAD

**Discussion by Perpixel** [mrcjkb/rustaceanvim#360] [Q&A]
I've been using rustaceanvim with lazy.nvim on macos (nvim 0.10.0) and can't get the cmp menu to respect the max width set in my global cmp configuration. I understand rustaceanvim take over, but not sure what to set to make this work correctly.

This is how it looks,

<img width="1440" alt="Screenshot 2024-04-15 at 8 48 52 AM" src="https://github.com/mrcjkb/rustaceanvim/assets/7267994/bfdff598-9b1e-4fa3-ae8f-858ec70d5015">
<img width="1421" alt="Screenshot 2024-04-15 at 8 48 25 AM" src="https://github.com/mrcjkb/rustaceanvim/assets/7267994/cad895a0-8ece-40d8-b181-05cbcc976263">

Any suggestion would be appreciated! thanks!

**Comment by mrcjkb** [comment]
Hey 👋 

Rustaceanvim doesn't take over nvim-cmp configuration.
